src/main.rs

```diff
@@ -187,5 +189,42 @@ async fn main() -> anyhow::Result<()> {
         "Starting Edgli"
     );
 
-    edgli::run_hopr_edge_node(cfg, hopr_keys).await
+    // TODO: not doing anything much, an edge node without the possibility of externally calling it.
+    //
+    // Pending decision on future interfaces (e.g. REST, gRPC,...)
+    let (_hopr, processes) = edgli::run_hopr_edge_node(cfg, hopr_keys).await?;
+
+    let mut signals =
+        Signals::new([Signal::Hup, Signal::Int]).map_err(|e| EdgliError::OsError(e.to_string()))?;
+    while let Some(Ok(signal)) = signals.next().await {
+        match signal {
+            Signal::Hup => {
+                info!("Received the HUP signal... not doing anything");
```

Comment:
SIGHUP should reload config file. 

Add this now and you won't have troubles with it later.